lib/resources/IssuerFraudRecords.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+var StripeResource = require('../StripeResource');
+var stripeMethod = StripeResource.method;
+
+module.exports = StripeResource.extend({
+  path: '',
+
+  list: stripeMethod({
+    method: 'GET',
+    path: 'issuer_fraud_records',
+  }),
+
+  retrieve: stripeMethod({
+    method: 'GET',
+    path: function(urlData) {
+      return 'issuer_fraud_records/' + urlData.issfrId;
+    },
+    urlParams: ['issfrId'],
+  }),
+
+  listFromCharge: stripeMethod({
+    method: 'GET',
+    path: function(urlData) {
+      return 'issuer_fraud_records?charge=' + urlData.chargeId;
+    },
+    urlParams: ['chargeId'],
+  }),
```

Comment:
Like in Ruby, `list` allows you to pass parameters to it. Here's an example from the subscriptions test suite:

``` js
      stripe.subscriptions.list({
        limit: 3,
        customer: 'test_cus',
        plan: 'gold',
      });
```